rcon/rcon.py

```diff
@@ -44,7 +44,9 @@
     ALL_ROLES,
     ALL_ROLES_KEY_INDEX_MAP,
     INDEFINITE_VIP_DATE,
+    _get_default_info_dict,
     get_server_number,
+    parse_raw_player_info,
```

Comment:
to make it consistent with my suggestion from above, what about renaming this one to:
`new_from_player_info`? :)

Reply:
I'm partial to this name, it says exactly what it does.

I did update the docstring though which I forgot to do.